pkg/defaultcomponents/defaults.go

```diff
@@ -72,6 +74,26 @@ import (
 	"go.uber.org/multierr"
 )
 
+var datadogExporterFeatureGate = featuregate.GlobalRegistry().MustRegister("adot.exporter.datadogexporter",
+	featuregate.StageBeta,
```

Comment:
Feature gates need to start at the alpha stage.  The name should probably be something like `adot.exporter.<name>.deprecation` so that the "enabled" state is removing the component.

Reply:
modified the gate with `deprecation` suffix.